README.md

```diff
@@ -4,9 +4,9 @@
 [![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
 [![CocoaPods](https://img.shields.io/cocoapods/v/MapboxNavigation.svg)](https://cocoapods.org/pods/MapboxNavigation/)
 
-<img alt="Mapbox Navigation SDK" src="https://github.com/mapbox/mapbox-navigation-ios/raw/master/docs/img/navigation.png" width="50%" style="margin: auto;display: block;" />
+<img alt="Mapbox Navigation SDK" src="./img/navigation.png" width="272" align="center" />
```

Comment:
Would it make sense to hardcode this URL?

Reply:
No, because this Markdown document is only viewed on GitHub now. If we decide to rename or move this image on a branch, each branch’s readme will continue to display a working image.

Reply:
Whoops, the URL was incorrect though: fixed in b3f3df7afd53be87bc4c3624dcaea108763ac2ca.